helm/featurehub/values.yaml

```diff
@@ -21,24 +101,134 @@ managementRepository:
     #   cpu: 100m
     #   memory: 128Mi
     # requests:
-  #   cpu: 100m
-  #   memory: 128Mi
+    #   cpu: 100m
+    #   memory: 128Mi
+
+  serviceAccount:
+    # Specifies whether a service account should be created
+    create: true
+    # Annotations to add to the service account
+    annotations: {}
+    # The name of the service account to use.
+    # If not set and create is true, a name is generated using the fullname template
+    name: ""
+
   nodeSelector: {}
+
   tolerations: []
+
   affinity: {}
 
+  # -- Environment variables to be exposed to management-repository deployment.
+  # As of 1.5.0 all FeatureHub controller properties are available as environment variables using the same case.
+  # Entries accepted here are the same as would be specified in /etc/app-config/applications.properties.
+  # Note that `server.port` and `monitor.port` use their default values of `8085` and `8701` respectively, to make
+  # it easier to implement the deployment, service and the prometheus serviceMonitor manifests.
+  environmentVars:
+    db.url: jdbc:postgresql://setmeup-postgresql-endpoint:5432/featurehub
+    db.username: featurehub
+    db.connections: "3"
+    run.nginx: "true"
+    server.gracePeriodInSeconds: "10"
+    register.url: http://localhost/register-url?token=%s
```

Comment:
ideally remove this as its no longer required for FH

Reply:
Sure @rvowles, let me quickly remove them if they are no longer needed.

Same as on the other comment, as I don't have all the knowledge on the application side, I tried collecting the required properties of each component just by having a look at the previous templates and the content of the files inside the Docker images.

Feel free to suggest more changes like this. In fact, I felt a bit lost while checking what properties apply to what component (MR, edge, or dacha), and the purpose of each of them. Not sure if because those are not documented or if I did not find them properly.

Regards!

Reply:
Suggested properties are now removed from the values.

Thanks!